src/Microsoft.AspNet.Server.Kestrel/Http/Frame.Templated.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Though files like https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging/LoggerOfT.cs do exist, I would actually suggest `Frame.OfT.cs` since that places it under `Frame.cs` under VS. Thoughts?